### ISSUE TITLE
tour: subtle/trivial bug in test

### ISSUE THREAD

**Issue by sfoley-gpqa** [golang/tour#1353]
Context: https://go.dev/tour/moretypes/23

In the test, one of the sentences is "the quick brown fox jumped over the lazy dog". The original sentence is designed to use every letter in the English alphabet; however, this one lacks the letter "s". The sentence is supposed to be "The quick brown fox jumps over the lazy dog"

**Comment by crisman** [comment]
Acknowledging the non-traditional sentence, but since the exercise does not do letter counts, how does that sentence cause any issues?

**Comment by ALTree** [comment]
Good catch but I think it's okay to leave the sentence as is to reduce code churn.
